SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialShadowActor.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+#include "SpatialConstants.h"
+#include "SpatialShadowActor.generated.h"
+
+DECLARE_LOG_CATEGORY_EXTERN(LogSpatialShadowActor, Log, All);
+
+UCLASS(Transient)
+class SPATIALGDK_API USpatialShadowActor : public UObject
```

Comment:
Does not need to be an UObject anymore?

Reply:
This class uses LOG facility from UObject and also now has a UProperty back again!